src/main/java/eutros/multiblocktweaker/gregtech/recipes/CustomMultiblockRecipeLogic.java

```diff
@@ -37,7 +38,6 @@ private void logFailure(String func, Throwable t) {
         CraftTweakerAPI.logError(String.format("Couldn't run %s function of %s.", func, getMetaTile().getMultiblock()), t);
     }
 
-    @Override
     protected int[] runOverclockingLogic(@NotNull Recipe recipe, boolean negativeEU, int maxOverclocks) {
```

Comment:
This signature should change to `IRecipePropertyStorage propertyStorage, int recipeEUt, long maxVoltage, int duration, int maxOverclocks` so that it maintains the override.